manifests/g/Geany/Geany-Plugins/1.38/Geany.Geany-Plugins.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Geany.Geany-Plugins
+PackageVersion: 1.38
+PackageLocale: en-US
+Publisher: The Geany contributors

```

```suggestion
Publisher: The Geany developer team
```
This must match Control Panel.